cyhy_report/customer/generate_report.py

```diff
@@ -2947,7 +2947,14 @@ def __generate_false_positives_attachment(self):
                 data_writer.writerow(row)
 
     def __generate_sub_org_summary_attachment(self):
-        if self.__snapshots[0].get("descendants_included"):
+        # If _all_ the descendants of an org have start dates in the
+        # future (i.e., are not yet scanning) then
+        # self.__snapshots[0].get("descendants_included") will be true
+        # but "ss0_descendant_data" will not be a key of
+        # self.__results.  We need to avoid this corner case.
+        #
+        # See issue #92 for more details.
+        if self.__snapshots[0].get("descendants_included") and "ss0_descendant_data" in self.__results:
```

Comment:
Now that I have thought about it, this is a little redundant.  I don't think there can be a situation where `"ss0_descendant_data" in self.__results` is true and `self.__snapshots[0].get("descendants_included")` is false, which means that this can be simplified to:
```suggestion
        if "ss0_descendant_data" in self.__results:
```

Thoughts?

Reply:
I'm fine with this.  What do you say, @felddy?

Reply:
Pretty sure it's all good, since the only way that `"ss0_descendant_data"` gets added is if there are descendant snapshots:
https://github.com/cisagov/cyhy-reports/blob/63b896a2e4cfe946b446cbfe034888a2e8a4fe8a/cyhy_report/customer/generate_report.py#L826-L859

Reply:
If @felddy doesn't answer by COB tomorrow then I'm fine proceeding without his imprimateur.

Reply:
We shall proceed without @felddy's imprimatur.